Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -145,6 +146,8 @@ public function search(SearchQuery $searchQuery)
                     'query' => $queryString,
                 ],
             ],
+            'from' => $searchQuery->getOffset(),
```

Comment:
how behaves this when offset and limit is the standard (i think `offset = 0; limit = null`)